todoapp/app/build.gradle

```diff
@@ -80,7 +80,7 @@ dependencies {
 
     // Android Testing Support Library's runner and rules
     androidTestCompile "com.android.support.test:runner:$rootProject.ext.runnerVersion"
-    androidTestCompile "com.android.support.test:rules:$rootProject.ext.runnerVersion"
+    androidTestCompile "com.android.support.test:rules:$rootProject.ext.rulesVersion"
```

Comment:
That's correct. I also feel the same, but I was not sure if these versions can be same always since they were mentioned separately, that is the reason I did not combine them into one.